libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -32,6 +32,8 @@ func newEc2MetadataFetcher(config *common.Config) (*metadataFetcher, error) {
 			"machine_type":      c.Str("instanceType"),
 			"region":            c.Str("region"),
 			"availability_zone": c.Str("availabilityZone"),
+			"accountId":         c.Str("accountId"),
+			"ami":               c.Str("imageId"),
```

Comment:
`ami` is not a vendor neutral term so I wouldn't want to put it into the schema shared by all of the provider implementations. `image.id` would be better in my opinion. This field will need to be added to the `_meta/fields.yml` that's in this package.

Reply:
@andrewkroh @webmat sounds like `cloud.image.id` would be a nice ECS candidate.
WDYT?

Reply:
@odacremolbap Interesting. I'll note it down as a thing to look into.

Volume snapshots & images are available on other technology platforms than cloud providers, however. So not sure the images will end up nested under cloud necessarily.

But you can still feel free to proceed with this for now. Beats has many non-ECS fields peppered inside other ECS field sets. A bit of a risk, but not the end of the world :-)

Reply:
@odacremolbap Best directly open an issue here to discuss: https://github.com/elastic/ecs